keras_cv/layers/object_detection/multi_class_non_max_suppression.py

```diff
@@ -43,7 +43,7 @@ def __init__(
         bounding_box_format,
         from_logits,
         iou_threshold=0.5,
-        confidence_threshold=0.5,
+        confidence_threshold=0.9,
```

Comment:
Is this default pretty specific to RetinaNet, or is it also a sensible default for FasterRCNN + potentially other models?

Reply:
From what I saw in @quantumalaviya 's work he ALSO tunes this to 0.9~ or so.  I think in YOLOx it may even be 0.99 actually.

In short: its probably not generically true, but its better than 0.5

Reply:
okay. Please update the docstring to reflect the new default.